Formula/docker.rb

```diff
@@ -18,8 +18,9 @@ class Docker < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "auto"
     dir = buildpath/"src/github.com/docker/cli"
-    dir.install (buildpath/"components/cli").children
+    dir.install (buildpath/"").children
```

Comment:
Does `dir.install buildpath.children` not work?

Reply:
Also, I actually suspect the correct solution here is to just adjust the flags to `system "go", "build"` below rather than to duplicate the contents of `buildpath` in `src/github.com/docker/cli`.

Reply:
@chenrui333 would like your thoughts on this.

Reply:
Yeah, I missed this, I think `dir.install buildpath.children` is better syntax.

But once docker move completely to go modules-based build, these setup can all be removed.

Reply:
log a ticket there, https://github.com/docker/cli/issues/2905

Reply:
I was thinking they could be removed right now by just adjusting the build flags, as it seems like it's the build flags that set up `go build` to look in `buildpath/src/github.com/docker/cli`. As long as we remember to fix it when docker uses go modules, then I guess it's fine. (I'm worried we'll forget though.)

Reply:
I'm not familiar with the syntax, so I "just" made it work. The change in repo made it necessary to get the right path.